### PR TITLE
Remove "Hybrid" from configurations displayed to user #1569

### LINKED ISSUE #1569: Remove "hybrid" from names in configuration tree
All hybrids in the configuration tree have the word "hybrid" which does not add useful information since they appear under the "Hybrid" bin name.

However, the configuration tree uses "hybrid" to identify hybrid technologies. Removing it requires reworking how hybrids are implemented in the user interface.

### PR DESCRIPTION
**Pull request by sjanzou**
SAM_1569 branch of SAM
image

### CODE REVIEW

File: api/include/SAM_Battery.h [hunk 1]
```diff
@@ -2399,6 +2399,8 @@ extern "C"
 
 	SAM_EXPORT double* SAM_Battery_Outputs_batt_dispatch_sched_mget(SAM_table ptr, int* nrows, int* ncols, SAM_error *err);
 
+	SAM_EXPORT double SAM_Battery_Outputs_batt_grid_charge_percent_nget(SAM_table ptr, SAM_error *err);
```
Comment (mjprilliman) [inline api/include/SAM_Battery.h:2402]: These outputs are from https://github.com/NREL/SAM/pull/1865 and https://github.com/NREL/ssc/pull/1215. Sorry for not updating the autogen!

File: api/api_autogen/library/defaults/Pvsamv1_PVBatterySingleOwner.json [hunk 2]
```diff
@@ -485,7 +485,8 @@
 		"Load": {
 		},
 		"PVLosses": {
-			"enable_subhourly_clipping": 0
+			"enable_subhourly_clipping": 0,
```
Comment (mjprilliman) [inline api/api_autogen/library/defaults/Pvsamv1_PVBatterySingleOwner.json:488]: These are from https://github.com/NREL/SAM/pull/1861.

File: api/modules/SAM_Pvwattsv8.cpp [hunk 3]
```diff
@@ -1253,6 +1253,16 @@ SAM_EXPORT double* SAM_Pvwattsv8_Outputs_snow_aget(SAM_table ptr, int* length, S
 	return result;
 }
 
+SAM_EXPORT double* SAM_Pvwattsv8_Outputs_snow_cover_aget(SAM_table ptr, int* length, SAM_error *err){
```
Comment (mjprilliman) [inline api/modules/SAM_Pvwattsv8.cpp:1256]: snow_cover added to pvwatts as part of https://github.com/NREL/ssc/pull/1214.

**Review by mjprilliman — APPROVED** [review]
Looks good. Thanks for updating the API for recent changes!

**Review by cpaulgilman — APPROVED** [review]
Configuration tree looks good without "Hybrid" appended to performance model names.

**Pull request merged**